grype/matcher/java/matcher.go

```diff
@@ -9,7 +16,104 @@ import (
 	syftPkg "github.com/anchore/syft/syft/pkg"
 )
 
+const (
+	sha1Query = `1:"%s"`
```

Comment:
I think I'm going to keep it here for now as a const to highlight how we're doing the maven queries. 

There might be a future where we include other query paths to lookup a package.  Keeping this easy to discover helps future readers find where the initial query is being executed.

Also giving it a title as `sha1Query` makes it easier to come back and see what's happening. Since this is happening in a very specific place this string is I'd rather it be self-documented than a magic inline query string.

Reply:
@wagoodman @luhring @jonasagx thoughts here?

Reply:
`sha1Query` will always be called as `fmt.Sprintf(sha1Query, sha1)`, maybe make that a function wrapping the query construction